tests/database.py

```diff
@@ -29,5 +30,5 @@ def get_task() -> asyncio.Task:
     return task
 
 
-TestSession = async_scoped_session(orm.sessionmaker(class_=AsyncSession, expire_on_commit=False), get_task)
+TestSession = async_scoped_session(async_sessionmaker(class_=AsyncSession, expire_on_commit=False, future=True), get_task)
```

Comment:
```suggestion
TestSession = async_scoped_session(async_sessionmaker(class_=AsyncSession, expire_on_commit=False), get_task)
```

`future=True` is only needed if `sqlalchemy<=2.0` but you want to use the 2.0 API style.